buildozer/targets/android.py

```diff
@@ -422,28 +422,23 @@ def _install_android_ndk(self):
         # from 10e on the URLs can be looked up at
         # https://developer.android.com/ndk/downloads/older_releases
 
+        is_darwin = platform in ('darwin', )
```

Comment:
I'm curious why we didn't simply make it an `==` initially, but it's not too late :smile: 
```suggestion
        is_darwin = platform == 'darwin'
```

Reply:
I wondered about that too :)